R/track.R

```diff
@@ -0,0 +1,30 @@
+#' Compute track statistics
+#'
+#' @param population ...
+#'
+#' @return track statistics
+#'
+#' @importFrom magrittr %>%
+#' @importFrom magrittr %<>%
+#' @export
+#'
+track <- function(population, group_track, ...) {
+  # process `population`, which is the data you get from CellProfiler
+  tracks <- add_dist(population, group_track)
+  
+  track_feature_list <- list(
```

Comment:
You’re protected by the function namespace you can use a clearer identifier like “features.”